### PR TITLE
Travisci sonarcloud take2

### PR DESCRIPTION
**Pull request by morgantaschuk**
Added TravisCI and sonarcloud to pinery.

### CODE REVIEW

File: pinery-client/src/main/java/ca/on/oicr/pinery/client/ProjectClient.java [hunk 1]
```diff
@@ -17,5 +17,6 @@ public ProjectClient(PineryClient mainClient) {
 	public List<SampleProjectDto> all() throws HttpResponseException {
 		return getResourceList("sample/projects");
 	}
-	
+    //TODO
```
Comment (sqwci) [inline pinery-client/src/main/java/ca/on/oicr/pinery/client/ProjectClient.java]: INFO Complete the task associated to this TODO comment. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)

File: pinery-client/src/main/java/ca/on/oicr/pinery/client/ProjectClient.java [hunk 2]
```diff
@@ -17,5 +17,6 @@ public ProjectClient(PineryClient mainClient) {
...
+    //FIXME
```
Comment (sqwci) [inline pinery-client/src/main/java/ca/on/oicr/pinery/client/ProjectClient.java]: MAJOR Take the required action to fix the issue indicated by this comment. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1134)

**Comment by sqwci** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by morgantaschuk** [comment]
Haha, I definitely will!

**Pull request merged**